index.js

```diff
@@ -58,7 +58,7 @@ const recontainerize = (snippetId, docker, indexContents, logging = true, lifeIn
 
 /* ---- API ---- */
 const queryDocumentCallback = (doc, logging = true, lifeInMilliseconds = 1000) => {
-  if (!doc.get('running')) return; // just to double check
+  //if (!doc.get('running')) return; // just to double check
```

Comment:
I'm assuming you commented this because you fixed the bug in the app. I think I still have the check for this in the Scala code.